pythonforandroid/recipes/hostpython3crystax/__init__.py

```diff
@@ -27,7 +27,6 @@ def build_arch(self, arch):
         Creates expected build and symlinks system Python version.
         """
         self.ctx.hostpython = '/usr/bin/false'
-        self.ctx.hostpgen = '/usr/bin/false'
         # creates the sub buildir (used by other recipes)
         # https://github.com/kivy/python-for-android/issues/1154
         sub_build_dir = join(self.get_build_dir(), 'build')
```

Comment:
We should probably rename the recipe class to `HostPython3CrystaxRecipe` so it doesn't "conflict" with the new one